### PR TITLE
Add a simple SparkExample

### PR DESCRIPTION
**Pull request by critical27**

### CODE REVIEW

**Comment by darionyaphet** [comment]
Thanks for your great work :) Could you support an example about the Spark example ?

**Comment by critical27** [comment]
> Thanks for your great work :) Could you support an example about the Spark example ?

Yeah, there is a simple example in `SparkExample.java`, which generate a JavaRDD from csv file, and do some simple work.

File: examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java [hunk 1]
```diff
@@ -149,6 +150,26 @@ public static void main(String[] args) {
                                                         value.columns.get(1).getInteger(),
                                                         new String(value.columns.get(2).getStr())));
             }
+
+            // Batch insert some edges for ScanEdgeInSpaceExample and SparkExample
+            for (long src = 1; src <= 10; src++) {
```
Comment (darionyaphet) [inline examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java]: Maybe we should have a test dataset. It's used by all the test case.

Comment (critical27) [reply examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java]: fixed

**Pull request merged**